subprojects/testing-jvm/src/main/java/org/gradle/api/internal/tasks/testing/JvmTestExecutionSpec.java

```diff
@@ -60,6 +66,16 @@ public JvmTestExecutionSpec(TestFramework testFramework, Iterable<? extends File
         this.javaForkOptions = javaForkOptions;
         this.maxParallelForks = maxParallelForks;
         this.previousFailedTestClasses = previousFailedTestClasses;
+        this.testIsModule = testIsModule;
+    }
+
+    @SuppressWarnings("unused")
+    @UsedByScanPlugin("test-retry")
+    public JvmTestExecutionSpec copyWithTestFramework(TestFramework testFramework) {
```

Comment:
Will need to update `test-retry` plugin to use this method

https://github.com/gradle/test-retry-gradle-plugin/blob/38d53f3a13d27685f5ca48e98a64f362071c1a8c/plugin/src/main/java/org/gradle/testretry/internal/executer/RetryTestExecuter.java#L143-L176